src/pages/ReimbursementAccount/USD/BusinessInfo/subSteps/NameBusiness.tsx

```diff
@@ -16,13 +16,17 @@ const STEP_FIELDS = [COMPANY_NAME_KEY];
 function NameBusiness({onNext, onMove, isEditing}: SubStepProps) {
     const {translate} = useLocalize();
 
-    const [reimbursementAccount] = useOnyx(ONYXKEYS.REIMBURSEMENT_ACCOUNT);
+    const [reimbursementAccount] = useOnyx(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {canBeMissing: true});
 
     const defaultCompanyName = reimbursementAccount?.achData?.companyName ?? '';
     const bankAccountID = reimbursementAccount?.achData?.bankAccountID;
 
     const bankAccountState = reimbursementAccount?.achData?.state ?? '';
-    const shouldDisableCompanyName = !!(bankAccountID && defaultCompanyName && ![BankAccount.STATE.SETUP, BankAccount.STATE.VERIFYING].includes(bankAccountState));
+    const shouldDisableCompanyName = !!(
+        bankAccountID &&
+        defaultCompanyName &&
+        ![CONST.BANK_ACCOUNT.STATE.SETUP, CONST.BANK_ACCOUNT.STATE.VERIFYING].includes(bankAccountState as typeof CONST.BANK_ACCOUNT.STATE.SETUP | typeof CONST.BANK_ACCOUNT.STATE.VERIFYING)
```

Comment:
```suggestion
        bankAccountState !== CONST.BANK_ACCOUNT.STATE.SETUP &&
        bankAccountState !== CONST.BANK_ACCOUNT.STATE.VERIFYING
```

I think this is simpler, which is same as the following file.